packages/server/src/services/chatflows/index.ts

```diff
@@ -198,6 +197,60 @@ const saveChatflow = async (newChatFlow: ChatFlow): Promise<any> => {
     }
 }
 
+const saveChatflows = async (newChatflows: Partial<ChatFlow>[]): Promise<any> => {
```

Comment:
should we rename to `importChatflows`? `saveChatflows` sounds like updating existing chatflows